.vscode/cspell.json

```diff
@@ -92,6 +101,14 @@
         "azure",
         "azurite"
       ]
+    },
+    {
+      "filename": "sdk/cosmosdb/azure_data_cosmos/**",
```

Comment:
Any reason not just `sdk/cosmosdb/**`? Precedent is also that we inline words instead of a separate dictionary file, but I'm open to it if @weshaggard or @hallipr don't have any problem with this. Still, I would suggest just `.dict.txt` or something so it doesn't show up by default in common dev shells. It's not something most people need to worry about, but the cSpell VSCode plug-in should still give them a choice to add words to it.

Reply:
The main reason I went with a custom dict here is that these are terms that will likely appear with some frequency in the Cosmos DB crate **and** are quite unique to Cosmos. For example `pkranges`, which refers to the Partition Key Ranges API. I'm certainly open to other options here though.

> Any reason not just sdk/cosmosdb/**

Nope, I'm happy to expand it up.

Reply:
Oh, and by inlining I didn't mean in the source itself but in the section you added here i.e., just a `"words": []` array instead of a dictionary. Again, though, I'm open to allowing for dictionary files within service directories if @weshaggard is cool with it. It provides isolation to just the service directory which shouldn't trigger such broad pipelines for modifying more generally-scoped files like we do for the same case in the specs repos.

Reply:
https://cspell.org/configuration/imports/

I've been starting to think we should just have service teams create a config in their directory and import the common shared one to avoid conflicts and allow customization.

Reply:
That said I'm also OK with just having a dictionary file in the service directory as well.

Reply:
Yeah, that's a good point. I'll stick with a `words` list here and we can add a dictionary file later if it gets to be a long list.